Jenkinsfile

```diff
@@ -10,7 +10,7 @@ pipeline {
     
     FROM_PR = "${env.BRANCH_NAME}".contains("PR-");
     
-    PRODUCT_BRANCH = "master"
```

Comment:
i think we can use here : FROM_PR ? env.CHANGE_TARGET : env.BRANCH_NAME.

Reply:
Yes we can. 
I discovered that the following syntax: `${env.BUILD_KEY == "master" ? X_VAR : Y_VAR}` works, this avoids using the `${env.FROM_PR}".replace("true", " ").replace("false", "-Psign");` pattern.

I will commit an updated version of the jenkins file in a future commit.